src/r2b2/tests/test_audit.py

```diff
@@ -1,14 +1,18 @@
 import pytest
 
+import r2b2.tests.util as util
 from r2b2.audit import Audit
 from r2b2.contest import Contest
 
+default_contest = util.generate_contest(100)
+
 
 class SimpleAudit(Audit):
     """For testing purposes only."""
```

Comment:
The creation of the new class was precisely the point of the test. Because the `Audit` class is an abstract base class, ensuring that a subclass can properly be created from `Audit`, invoke `super().__init__()`, and create unique implementations of the abstract functions seemed like an important test.

Reply:
Ahh - noting that reasoning may be all we need.
But wouldn't we get more meaningful testing coverage of that init code for free when testing the other classes we derive from Audit?